### ISSUE TITLE
SyntaxError

### ISSUE THREAD

**Issue by nicobubulle** [nosmokingbandit/Watcher3#110]
Hello,

I have a syntax error when I try to launch watcher : 
```
Traceback (most recent call last):
  File "watcher.py", line 107, in <module>
    from core.app import App
  File "/opt/watcher/core/app.py", line 108
    return App.status_template.render(**self.defaults(), profiles=core.CONFIG['Quality']['Profiles'].keys(), movie_count=movie_count)
                                                       ^
SyntaxError: invalid syntax
```

It' seems to be related to commit da880c919335db273620d244a5e56b98a8180c41

If i use previous commit it work fine.

**Comment by nosmokingbandit** [comment]
02fe5267cceae78edb498ba924faf53205773d8f

I'd guess you are using python 3.4? I was able to reproduce this in 3.4 but not 3.6, but this commit fixes 3.4. Seems like certain earlier versions of python 3 require args to be passed more specifically.

In other words: It should be fixed, let me know if it isn't.

**Comment by nicobubulle** [comment]
Yes i'm using python 3.4. error have been solved.

Thanks
